resources/outputs.js

```diff
@@ -88,7 +88,7 @@ const Outputs = new ThrowOnInvalidProxy({
     de: 'Tank buster',
     fr: 'Tank busters',
     ja: 'タンクバスター',
-    cn: '死刑',
+    cn: '坦克死刑',
```

Comment:
These are multiple tankbusters, but there are no plural form in JA/CN, how can we call them?